relay-server/src/services/processor.rs

```diff
@@ -234,6 +234,15 @@ impl ProcessingGroup {
             ))
         }
 
+        let is_metric = |i: &Item| matches!(i.ty(), ItemType::Statsd | ItemType::MetricBuckets);
```

Comment:
Might be worth adding metrics meta to the integration test as well:

```suggestion
        let is_metric = |i: &Item| matches!(i.ty(), ItemType::Statsd | ItemType::MetricBuckets | ItemType::MetricMeta);
```

We could add a `is_metrics()` method to `ItemType` and use it in `queue_envelope` and here.